test/observable/variance/test_variance.py

```diff
@@ -0,0 +1,129 @@
+import netket as nk
```

Comment:
just move this file inside of test/observable/test_variance.py , no need to create an extra folder.